src/components/Avatar/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { View,Text } from 'react-native';
+import { Loading } from '../Loading';
+import { useAuthStore } from '../../shared/zustand/auth';
+import { THEME_COLORS, THEME_FONTS } from '../theme';
+import { styles } from './style';
+
+
+const Avartar = ({
+    type = "circle",
+    avatarUri,
+    width,
+    height,
+    content,
+}) => {
+    const { isLoading } = useAuthStore();
```

Comment:
generic components should not interact with any Zustand stores directly. Change `isLoading` to a prop of this `Avatar` component and pass the value from the parent screen component